docs/initialization-steps-and-the-related-tracer.org

```diff
@@ -0,0 +1,624 @@
+#+AUTHOR: Pawel Szulc
+#+TITLE: CAD-3564 Catalogue node initialization steps and the related tracer events that already exist
+#+LaTeX_CLASS: article
+#+LATEX_CLASS_OPTIONS: [a4paper,10pt]
+#+LATEX_HEADER: \usepackage[margin=0.9in]{geometry}
+#+OPTIONS: toc:nil
+
+#+begin_export latex
+  \tableofcontents \clearpage
+#+end_export
+Consensus entry point for Node is ~Ouroboros.Consensus.Node~ and its function
+~runWith~ - this function starts a consensus. It will:
++ open Chain DB
+  + open ImmutableDb
+  + open VolatileDB
+  + open LedgerDB
++ setup the NodeKernel
++ initialize the network layer
+
+This document focuses solely on the initialization steps of opening ChainDB. The
```

Comment:
Maybe we should limit the line length to 80 chars.

Reply:
<img width="649" alt="Screenshot 2021-11-23 at 12 17 52" src="https://user-images.githubusercontent.com/6287558/143015090-671ea299-ed43-4bfd-b6db-1c0d33aa8f5f.png">

It's set to 80. Always has been :)